docs/cmd/kn_service_update.md

```diff
@@ -41,6 +41,7 @@ kn service update NAME [flags]
   -p, --port int32               The port where application listens on.
       --requests-cpu string      The requested CPU (e.g., 250m).
       --requests-memory string   The requested memory (e.g., 64Mi).
+      --revision-name string     The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
```

Comment:
Since I suspect this flag will be used a lot, what do you think about having it available via `-r` too ?

Reply:
Easy enough to do later.  Let's do the simpler thing for now — I'm nervous-er about doing shortopts w/o consensus much more than longopts.

Reply:
Now that I know the service name prefix is mandatory for a revision, I wonder whether we can help the user to do this for her automatically if not already existant? E.g. allowing for name `{{.Random 5}}-{{.Generation}}` (which would be a good benefit because of the verboseness of Go templates).

If not, we should at least add client-side validation for this constraint.